tests/arp/test_tagged_arp.py

```diff
@@ -149,4 +149,15 @@ def test_tagged_arp_pkt(ptfadapter, duthosts, rand_one_dut_hostname,
                 assert arp_cnt == DUMMY_ARP_COUNT, "Expect {} entries, but {} found".format(DUMMY_ARP_COUNT, arp_cnt)
             except Exception as detail:
                 logger.error("Except: {}".format(detail))
+                # Dump status for debug
```

Comment:
Is it a general infra requirement? if a critical testcase failed, run as much as dump functions like `show tech`, and stop running other testcases.